src/cirrocumulus/scripts/cirro_scanner.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# cirro_scanner.sh scans the mounted buckets to find cirro data. Creates a cirro dataset when *.zarr folder
+# is found and is not yet in the existing cirro datasets.
+
+if [[ $# -ne 1 ]]; then
+  echo "Usage: $0 <true/false>"
+  exit 1
+fi
+
+readonly LOG_IN="${1}"
+if [[ "${LOG_IN}" == "false" ]]; then
```

Comment:
This is an unusual usage pattern. I don't really think that the script should exit with a clean exit code (0) if it didn't do anything.

If the user is not logged in, then does "wb resource mount" fail in a way that makes it clear? Alternatively, it feels simpler to just document that this script should only be called when the user is already logged in to Workbench (clearly the caller already knows). If there's a common usage where the script is called without a user logged in already _and_ that usage needs to exit cleanly (exit 0), as the behavior of this script is unexpected.

Reply:
ok i will document it then